base/docs/basedocs.jl

```diff
@@ -276,6 +276,19 @@ julia> z
 """
 kw"global"
 
+"""
+    ' '
+
+the single-quote character delimits Char (that is, character) literals.
+
+# Examples
+```jldoctest
+julia> println('j')
+j
+```
+"""
+kw"' '"
```

Comment:
I'd suggest making this just two literal quotes, since ' ' already prints something, and then add something to `kw"'"` also to suggest looking here. Something like:

````suggestion
A pair of single-quote characters delimit a [`Char`](@ref) (that is, character) literal.

# Examples
```jldoctest
julia> 'j'
'j': ASCII/Unicode U+006A (category Ll: Letter, lowercase)
```
"""
kw"''"
````